src/ResourceManager/Sql/Commands.Sql/packages.config

```diff
@@ -1,5 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Azure.Management.Sql" version="1.13.0-preview" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.Sql" version="1.15.0-preview" targetFramework="net452" />
   <package id="Microsoft.Azure.Management.Storage" version="2.4.0-preview" targetFramework="net45" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.11" targetFramework="net452" />
```

Comment:
@payiAzure please remove the added references in this file

Reply:
@cormacpayne will remove the added files in the next commit.